mmcv/runner/checkpoint.py

```diff
@@ -260,7 +260,7 @@ def load_from_local(filename, map_location):
     Returns:
         dict or OrderedDict: The loaded checkpoint.
     """
-
+    filename = osp.expanduser(filename)
```

Comment:
line #265 can be changed to `raise FileNotFoundError(f'{filename} can not be found.')`

Reply:
Done